libs/tiled-bitmap/src/cmyklayeroperations.cc

```diff
@@ -0,0 +1,331 @@
+/*
+ * Scroom - Generic viewer for 2D data
+ * Copyright (C) 2009-2020 Kees-Jan Dijkzeul
+ *
+ * SPDX-License-Identifier: LGPL-2.1
+ */
+
+#include <scroom/layeroperations.hh>
+
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+
+#include <glib.h>
+
+#include <sstream>
+
+#include <boost/shared_ptr.hpp>
+#include <boost/utility.hpp>
+
+#include <scroom/bitmap-helpers.hh>
+#include <scroom/cairo-helpers.hh>
+#include <scroom/unused.hh>
+
+#include <boost/math/special_functions/round.hpp>
```

Comment:
Where's this one used?

Reply:
See the third bullet point in the answer to your second comment on this file.

Reply:
Good catch. I copied the includes from `layeroperations.cc` without properly checking that all of them were still required. I pushed a new commit that removed all include statements that are not needed.